state/factory/workingset.go

```diff
@@ -333,7 +333,7 @@ func (ws *workingSet) Commit(ctx context.Context) error {
 		return err
 	}
 	ws.Reset()
-	return nil
+	return ws.store.Stop(ctx)
```

Comment:
why add this?

Reply:
newWorkingSet() creates a `factoryWorkingSetStore` instance and calls `Start()` inside, but its `Stop()` is never called.

Reply:
After checking, `workingSet.Commit()` calls `store.tlt.RootHash()`, which calls `tlt.flush()`, which is the effective Stop(), we may need another PR to clean-up this